tests/integration/cartography/intel/aws/test_rds.py

```diff
@@ -1,16 +1,63 @@
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
 import cartography.intel.aws.rds
+from cartography.intel.aws.rds import sync
 from tests.data.aws.rds import DESCRIBE_DBCLUSTERS_RESPONSE
 from tests.data.aws.rds import DESCRIBE_DBINSTANCES_RESPONSE
 from tests.data.aws.rds import DESCRIBE_DBSNAPSHOTS_RESPONSE
+from tests.integration.cartography.intel.aws.common import create_test_account
+from tests.integration.util import check_nodes
+from tests.integration.util import check_rels
 
+TEST_ACCOUNT_ID = "000000000000"
+TEST_REGION = "us-east-1"
 TEST_UPDATE_TAG = 123456789
 
 
+def _create_test_security_groups(neo4j_session):
+    """
+    Create the EC2 security groups that are referenced in the RDS test data
+    """
+    security_group_ids = ["sg-some-othersg", "sg-some-sg", "sg-secgroup"]
+    for sg_id in security_group_ids:
+        neo4j_session.run(
+            """
+            MERGE (sg:EC2SecurityGroup{id: $sg_id})
+            ON CREATE SET sg.firstseen = timestamp()
+            SET sg.lastupdated = $update_tag
+            """,
+            sg_id=sg_id,
+            update_tag=TEST_UPDATE_TAG,
+        )
+
+
+def _create_test_subnets(neo4j_session):
+    """
+    Create the EC2 subnets that are referenced in the RDS test data
+    """
+    subnet_ids = ["subnet-abcd", "subnet-3421", "subnet-4567", "subnet-1234"]
+    for subnet_id in subnet_ids:
+        neo4j_session.run(
+            """
+            MERGE (subnet:EC2Subnet{subnetid: $subnet_id})
+            ON CREATE SET subnet.firstseen = timestamp()
+            SET subnet.lastupdated = $update_tag
+            """,
+            subnet_id=subnet_id,
+            update_tag=TEST_UPDATE_TAG,
+        )
+
+
 def test_load_rds_clusters_basic(neo4j_session):
```

Comment:
I made sure that the existing tests keep working. Just needed to add a transform